projects/node-api-test/package.json

```diff
@@ -16,6 +16,7 @@
   "author": "Graphistry, Inc.",
   "license": "Apache-2.0",
   "dependencies": {
-    "@graphistry/node-api": "^4.4.1"
+    "@graphistry/node-api": "^4.4.1",
+    "apache-arrow": "^11.0.0"
```

Comment:
[RAPIDS 23.02](https://github.com/graphistry/graphistry/blob/master/compose/dockerfiles/base/05-nvidia.Dockerfile#L15) is using [Arrow 10.x](https://github.com/rapidsai/cudf/blob/branch-23.02/conda/recipes/libcudf/conda_build_config.yaml#L20)
I don't know if is super critical to keep the same arrow version across all our stack, but oftentimes I found that it's a good idea to do so.

Reply:
I think we're fine wrt client, b/c format should be stable and this is about a js client -> our python

But once the server is upgraded (tonight) with a fix around categorical uploads, I can play with doing a lower minimum apache-arrow here

Reply:
Got it!